odl/discr/grid.py

```diff
@@ -673,50 +575,8 @@ def meshgrid(self, squeeze=False):
         >>> x**2 - y**2
         array([[-1.,  0., -4.],
                [ 0.,  1., -3.]])
-
-        Can return 1d arrays by using squeeze
-
-        >>> x, y = g.meshgrid(squeeze=True)
-        >>> x
-        array([ 0., 1.])
-        >>> y
-        array([-1.,  0.,  2.])
-
-        Fortran ordering of the grid is respected:
-
-        >>> g = TensorGrid([0, 1], [-1, 0, 2], order='F')
-        >>> x, y = g.meshgrid()
-        >>> x.flags.f_contiguous, y.flags.f_contiguous
-        (True, True)
-
-        """
-        return sparse_meshgrid(*self.coord_vectors,
-                               squeeze=squeeze, order=self.order)
-
-    def convex_hull(self):
```

Comment:
OK, but you can always use `partition.bbox` which gives the "correct" hull, this one would end at the min/max grid points.